docs/dev/run_the_etl.rst

```diff
@@ -200,13 +334,30 @@ years.
 Additional Notes
 ----------------
 The commands above should result in a bunch of Python :mod:`logging` output
-describing what the script is doing, and file outputs in the ``sqlite``,  and
-``parquet`` directories within your workspace. When the ETL is complete, you
-should see new files at ``sqlite/ferc1.sqlite`` and ``sqlite/pudl.sqlite`` as
-well as a new directory at ``parquet/epacems`` containing nested directories
-named by year and state.
-
-If you need to re-run ``ferc_to_sqlite`` or ``pudl_etl`` and want to overwrite
-their previous outputs you can add ``--clobber`` (run ``script_name --clobber``).
+describing what the script is doing, and file outputs in the ``outputs``
+directory within your workspace. When the ETL is complete, you
+should see new files at ``outputs/ferc1.sqlite`` and ``outputs/pudl.sqlite`` as
+well as a new directory at ``outputs/hourly_emissions_epacems`` containing
+nested directories named by year and state.
+
+If you need to re-run ``ferc_to_sqlite`` and want to overwrite
+their previous outputs you can add ``--clobber`` (run ``ferc_to_sqlite --clobber``).
 All of the PUDL scripts also have help messages if you want additional information
 (run ``script_name --help``).
+
+.. note::
+
+  The ``pudl_etl`` command does not have a ``--clobber`` option because
+  each etl run uses the same database file to read and write tables.
+  This enables re-running portions of the ETL.
+
+Foreign Keys
+------------
+Assets are loaded into the ``pudl.sqlite`` in parallel so foreign key constraints
+can not be evaluated in real time. However, foreign key constraints can be evaluated
+after all of the data has been loaded into the database. To check the constraints,
+run:
+
+```
+check_pudl_fks
+```
```

Comment:
This should probably look like CLI code and have the $ etc with `.. code-block:: console`

Reply:
Thanks! updated